### DISCUSSION TITLE
News Cache [2023-05-01]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#2255] [Announcements]
### 📣 Announcement

[Vercel Ship](https://vercel.com/ship) started today! You'll find a new video at [vercel.com/ship](https://vercel.com/ship) every day this week, with a live Keynote May 4th at 9am PST

### 🚢  Ships

- [Storage on Vercel](https://vercel.com/blog/vercel-storage)
    - [Vercel KV](https://vercel.com/changelog/vercel-kv)
    - [Vercel Postgres](https://vercel.com/changelog/vercel-postgres)
    - [Vercel Blob](https://vercel.com/changelog/vercel-blob)
- [Inspect and validate Open Graph metadata for enhanced link sharing](https://vercel.com/changelog/open-graph-link-sharing-inspector)
- [Detailed deployment summaries](https://vercel.com/changelog/detailed-deployment-summaries)

**Comment by amyegan** [comment]
Day Two: [Vercel Firewall and Vercel Secure Compute](https://vercel.com/blog/vercel-security)

**Comment by amyegan** [comment]
Day Three: [Vercel Spaces](https://t.co/Qh7hU0aQ57) and [Visual Editing](https://vercel.com/blog/visual-editing)

**Reply by alistairholt** [reply]
Will Visual Editing eventually be available to non-enterprise teams?

**Comment by amyegan** [comment]
Day Four: [Next.js 13.4](https://vercel.com/changelog/next-js-13-4) and [Vercel Data Cache](https://vercel.com/changelog/introducing-vercel-data-cache)

**Comment by amyegan** [comment]
Day Five: Celebrating the ecosystem, featuring [Auth with Clerk](https://vercel.com/blog/authentication-for-the-frontend-cloud) and [improved support for Nuxt on Vercel](https://vercel.com/blog/nuxt-on-vercel)
